microsoftgraph/microsoft-graph-explorer-v4
# Fix: Add required header for tip annotations display

## Overview

Fixes #2636 

### Demo

![image](https://github.com/microsoftgraph/microsoft-graph-explorer-v4/assets/18407044/44768d17-5c2b-478e-84ff-b17f2d8bb22f)

### Notes

- Works for signed-in users 
- Doesn't affect queries with no tip message 

## Testing Instructions

* Check out this branch and load Graph Explorer locally
* Sign in with the preferred account
* Run `/users` query
* Observe tip message
